planning/scenario_planning/scenario_selector/launch/dummy_scenario_selector_lane_driving.launch.xml

```diff
@@ -9,6 +9,10 @@
   <arg name="input_twist" default="" />
   <arg name="is_parking_completed" default="" />
 
-  <node pkg="topic_tools" type="relay" name="scenario_trajectory_relay" args="$(arg input_lane_driving_trajectory) $(arg output_trajectory)" output="screen"/>
-  <node pkg="rostopic" type="rostopic" name="scenario_pub" args="pub $(arg output_scenario) autoware_planning_msgs/Scenario '{current_scenario: LaneDriving, activating_scenarios: [LaneDriving]}'" output="screen"/>
+  <node pkg="topic_tools" exec="relay" name="scenario_trajectory_relay" output="log" >
+    <param name="input_topic" value="$(var input_lane_driving_trajectory)" />
+    <param name="output_topic" value="$(var output_trajectory)" />
+    <param name="type" value="autoware_planning_msgs/msg/Trajectory" />
+  </node>
+  <!-- <node pkg="ros2topic" type="ros2topic" name="scenario_pub" args="pub $(var output_scenario) autoware_planning_msgs/msg/Scenario '{current_scenario: LaneDriving, activating_scenarios: [LaneDriving]}'" output="screen"/> -->
```

Comment:
I played out a little bit and i think the following should work given that you add `ros2cli` as a dependency:

```xml
  <executable cmd="ros2 topic pub $(var output_scenario) autoware_planning_msgs/msg/Scenario '{current_scenario: Parking, activating_scenarios: [Parking]}'"
              name="scenario_pub" output="screen" shell="true"/>
```

Reply:
So I tried this and added the line to the dummy launch files here: fa49bfd

However when running these packages with the following command I get an error: 
```
INFO] [scenario_pub-2]: process started with pid [4947]
[scenario_pub-2] usage: ros2 [-h] Call `ros2 <command> -h` for more detailed usage. ...
[scenario_pub-2] ros2: error: unrecognized arguments: Parking, activating_scenarios: [Parking]}
[ERROR] [scenario_pub-2]: process has died [pid 4947, exit code 2, cmd 'ros2 topic pub /test_out_scenario autoware_planning_msgs/msg/Scenario {current_scenario: Parking, activating_scenarios: [Parking]}'].
```

Not too sure what this might indicate but running the exact same command directly through the command line runs the command line publisher correctly, but through ros2 launch this seems to fail. 

So then I did some experimenting and found that by simply moving the value of `cmd` into its own argument as so:
```
<arg name="cmd" default="ros2 topic pub $(var output_scenario) autoware_planning_msgs/msg/Scenario '{current_scenario: Parking, activating_scenarios: [Parking]}'" />
<executable cmd="$(var cmd)" name="scenario_pub" output="screen" shell="true"/>
```

This way I was able to successfully publish to the topic. I don't really have an explanation for why this is the case. Changes here: c0b6d61

Reply:
That's strange, I think i could actually launch it directly but I don't have a problem with this state as well.